gulpfile.js

```diff
@@ -954,11 +954,10 @@ function compileJs(srcDir, srcFilename, destDir, options) {
     const startTime = Date.now();
     return toPromise(bundler.bundle()
       .on('error', function(err) {
-        if (err instanceof SyntaxError) {
-          console.error(red('Syntax error: ' + err.message));
-        } else {
-          console.error(red(err.message));
-        }
+        // Drop the node_modules call stack, which begins with '    at'.
+        const message = err.stack.replace(/    at[^]*/, '').trim();
```

Comment:
why do we want to drop it?

Reply:
The stack has nothing to do with the actual syntax error, and gets cumbersome to read. For example, here's what I see when I print the entire stack:
```
SyntaxError: /usr/local/google/home/rsimha/src/amphtml/src/service-worker/kill.js: Unexpected token (28:6)
  26 | });
  27 | if (foo)
> 28 |   bar(;
     |       ^
  29 | 
  30 | self.addEventListener('activate', function(event) {
  31 |   // Usually after waiting for a navigation, it has to wait for all other
    at Parser.pp$5.raise (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:4454:13)
    at Parser.pp.unexpected (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:1761:8)
    at Parser.pp$3.parseExprAtom (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3750:12)
    at Parser.pp$3.parseExprSubscripts (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3494:19)
    at Parser.pp$3.parseMaybeUnary (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3344:19)
    at Parser.pp$3.parseExprListItem (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:4312:16)
    at Parser.pp$3.parseCallExpressionArguments (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3573:20)
    at Parser.pp$3.parseSubscripts (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3533:31)
    at Parser.pp$3.parseExprSubscripts (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3504:15)
    at Parser.pp$3.parseMaybeUnary (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3474:19)
    at Parser.pp$3.parseExprOps (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3404:19)
    at Parser.pp$3.parseMaybeConditional (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3381:19)
    at Parser.pp$3.parseMaybeAssign (/usr/local/google/home/rsimha/src/amphtml/node_modules/babylon/lib/index.js:3344:19)
```